### DISCUSSION TITLE
Update über CLI

### DISCUSSION THREAD

**Discussion by blackbasket** [OpenCCU/OpenCCU#2610] [🇩🇪 Fragen&Antworten]
Hallo zusammen,

gibt es eine Möglichkeit, wie ich die RM über CLI updaten kann?

Hintergrund ist, dass ich eine entfernte RM über den CloudMatic-Zugriff nicht updaten kann. Egal, ob ich das File uploade oder direkt auf die RM downloade, funktioniert das Update nicht. Es erscheint einfach nicht der Dialog, in dem man das Update dann startet.

LG,
Marcel

**Comment by Baxxy13** [comment]
Ja sicher, mache ich bei meinen Test-Systemen nur noch so.
auf der Konsole mal:
`/bin/checkFirmwareUpdate.sh -h`

Den Rest findest du dank der Hilfe die erscheint selbst raus. 😉

**Reply by blackbasket** [reply]
Danke :) werde ich die Tage testen ;)
